scripts/build_locally.py

```diff
@@ -70,6 +67,14 @@ def run(
         dest="oneapi",
         action="store_true",
     )
+    driver.add_argument(
```

Comment:
Can we use `choices` keyword for `add_argument` here with default to "Release"?

Reply:
Yes, I will add a list of allowable values for `--debug` argument.